tests/Makefile

```diff
@@ -9,7 +9,7 @@
 export TESTDIR?=${CURDIR}
 OUTPUTDIR=${TESTDIR}/test-output
 TESTS=eth0-basic secondary-basic secondary-more-ips
-SCRIPTS=ec2ifup ec2dhcp.sh ec2ifdown
+SCRIPTS=ec2ifup ec2dhcp-up-ipv4.sh ec2dhcp-up-ipv6.sh ec2dhcp-down-ipv4.sh ec2dhcp-down-ipv6.sh ec2ifdown
```

Comment:
`ec2dhcp.sh` is no longer in this list but still exists. Is this by design?

Reply:
no, it should have been deleted. fixed.

Reply:
Spoke with nmeyerhans to clear up my confusions here.